ml-agents/setup.py

```diff
@@ -37,7 +37,8 @@
         'docopt',
         'pyyaml',
         'protobuf>=3.6,<3.7',
-        'grpcio>=1.11.0,<1.12.0'],
+        'grpcio>=1.11.0,<1.12.0',
+        'pypiwin32==223;platform_system=="Windows"'],
```

Comment:
Was this tested on both Mac and Windows ?

Reply:
Yes I tested this on Mac, Windows and Linux.